integrations/docker/images/vscode/chip-build-vscode/Dockerfile

```diff
@@ -106,6 +106,16 @@ RUN set -x \
     telnet \
     srecord \
     openjdk-8-jdk \
+    git vim emacs sudo \
```

Comment:
There is a comment above that explained the previous choices.

We need to explain these additional ones as well. The vscode image is meant for `build` at this point but not as a full fledged development environment. 

Beyond that, any changes in Dockerfiles must be acompanied by a version bump. However overall I do not think we should have many of these items here (and if we do it seems we want them grouped/ordered and one per line for clear updates if we ever update them)

Reply:
Makes sense. I reverted these apt installs and removed unneeded ones.